packages/flame/lib/camera.dart

```diff
@@ -0,0 +1,12 @@
+export 'src/camera/behaviors/bounded_position_behavior.dart'
```

Comment:
Hmm, by creating this barrel file it is no longer clear to the user that the camera component is in the experimental stage, seems like a change outside of the scope of this PR?

Reply:
Ok, I've removed the export file.